metadata-ingestion/docs/transformer/dataset_transformer.md

```diff
@@ -1160,6 +1160,9 @@ setup(
 )
 ```
 
+Additionally, declare the transformer under the `entry_points` variable of the [setup script](../../setup.py). This enables the transformer to be
```

Comment:
In conjunction to this, could you update the transformer setup example's setup.py file https://github.com/datahub-project/datahub/blob/a1d1dd42691e87eef6cd53db88d162cde5c6540e/metadata-ingestion/examples/transforms/setup.py#L6

It'd also probably be clearer to simply inline the `datahub.ingestion.transformer.plugins` value instead of referencing setup.py

Reply:
You're right, I have simply copied these two sentences from [here](https://datahubproject.io/docs/metadata-ingestion/adding-source/#5-enable-discoverability), but directly giving an example might be a bit better. 😁